v0/destinations/serenytics/utils.js

```diff
@@ -0,0 +1,64 @@
+const set = require("set-value");
+const {
+  constructPayload,
+  extractCustomFields,
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  CustomError
+} = require("../../util");
+const { MAPPING_CONFIG } = require("./config");
+
+const responseBuilder = (STORAGE_URL, payload, messageType) => {
+  if (!STORAGE_URL) {
+    throw new CustomError(
+      `[Serenytics]: storage url for "${messageType.toUpperCase()}" is required.`,
+      400
+    );
+  }
```

Comment:
We can also put a regex validation for valid storage_url.

Reply:
updated